src/reactjs/components/ProfileEditor/ProfileActionFlyout.tsx

```diff
@@ -272,42 +272,63 @@ profileFlyoutState
         isFalsy(this.state.profileFormDetails.amtPassword) &&
           !isFalsy(this.state.profileFormDetails.generateRandomPassword)
           ? null
-          : parseInt(this.state.profileFormDetails.randomPasswordLength),
+          : parseInt(this.state.profileFormDetails.passwordLength),
       mebxPasswordLength:
       isFalsy(this.state.profileFormDetails.mebxPassword) &&
           !isFalsy(this.state.profileFormDetails.generateRandomMEBxPassword)
         ? null
-        : parseInt(this.state.profileFormDetails.randomMEBXPasswordLength),
+        : parseInt(this.state.profileFormDetails.mebxPasswordLength),
       ciraConfigName: this.state.profileFormDetails.networkConfigName !== 'dhcp_disabled'
         ? this.state.profileFormDetails.ciraConfigName
         : ''
     }
-    const payload = this.removeUnAssignedProperties(this.removeMEBxFieldsforCCM(obj))
+    let payload = this.removeUnAssignedProperties(this.removeMEBxFieldsforCCM(obj))
     const server: string = this.props.rpsServer != null ? this.props.rpsServer : ''
+    const cleanObj = (payload): any => {
+      for (const propName in payload) {
+        console.log('propName:', propName)
```

Comment:
do we need to keep the console.logs? I think we should remove them all

Reply:
done